### PR TITLE
Navigation menu

### PR DESCRIPTION
**Pull request by farzanehfar**
Resolves: #49 

- [x] Assigned a Reviewer to the **PR**.
- [x] Assigned a Project to the **PR**.
- [ ] Added **tests** for changed code.

## What Changed and Why?

  1. Added a placeholder for containers
  2. Added settings with inputId and languages
  3. Added Drop down menu placeholder
  4. Added plus for devices  
  5. Added stacks for tab navigations

## How to Review Changes?

  1. https://xd.adobe.com/view/d8cb1f48-e69d-4acb-b97d-9ec34c15f0b7-8d98/screen/a5b04331-a6a9-4f14-ab3a-35b506065df6

### CODE REVIEW

File: src/containers/LanguageSelection.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  const setLanguage = language => {
+    setLang(language.code);
+    showhideModal();
+  };
+
+  const languageButtons = (
+    <RadioButtons options={languages} active={lang} onPress={setLanguage} />
+  );
+
+  let activeLanguage = languages.filter(l => l.code === lang)[0].name;
```
Comment (drewmiley) [inline src/containers/LanguageSelection.js:43]: This could be a const or just set it as the content prop?

File: src/stacks/contributions.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        backgroundColor: Colors.PRIMARY,
+      },
+      headerTintColor: Colors.WHITE,
+    }}>
+    <StackScreen
+      name="Contributions"
+      component={Contributions}
+      options={{
+        title: 'Contributions',
+        headerTitleStyle: {
```
Comment (drewmiley) [inline src/stacks/contributions.js:27]: Could/should this be moved into styles folder?

File: src/components/SettingsRow.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import {StyleSheet, Text, TouchableHighlight, View} from 'react-native';
+import {Colors, Typography, Spacing} from '../styles';
+
+const SettingsRow: () => React$Node = ({title, content, onPress = null}) => {
```
Comment (drewmiley) [inline src/components/SettingsRow.js:5]: Should we make content -> children?

File: src/stacks/contributions.js (hunk 2 above)
Comment (jawrainey) [reply src/stacks/contributions.js:27]: Great point -- maybe add `/styles/.stack.js` and simply have the `headerTitleStack` variable in there?

**Comment by drewmiley** [comment]
This works locally for me 👍

**Pull request merged**